### PR TITLE
Clean more stale origins

### PR DESCRIPTION
**Pull request by originalsouth**
### Changes

Clean up all non-inference type origins that have empty results in Octpoes/XTDB.

### Issue link

N/A

### Demo

N/A

### QA notes

Verify that, in a steady state, there are no non-inference type origins that have empty results in Octpoes/XTDB. This can be done using xtdb-cli.py or visually VisualOctopoesStudio.

To use xtdb-cli.py the following query checks for empty results for non-inference origins:
`octopoes/tools/xtdb-cli.py query '{:query {:find [(pull ?var [*])] :where [[?var :type "Origin"]]}}}' | jq -C '.[][] | select(.origin_type != "inference" and (.result | length == 0))'`

---

### Code Checklist

- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] I have checked the documentation and made changes where necessary.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

- [x] Tickets have been created for newly discovered issues.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Comment by noamblitz** [comment]
Will test with empty observations in a bit :)

**Comment by noamblitz** [comment]
Jup works. Normalizers that yield empty observations after yielding observations with objects still nicely clean up their old data.

**Comment by noamblitz** [comment]
image

After running ssl certificates on a website without a cert, the observation is nomo

File: octopoes/octopoes/core/service.py [hunk 1]
```diff
@@ -191,6 +191,9 @@ def save_origin(
             self.ooi_repository.save(ooi, valid_time=valid_time, end_valid_time=end_valid_time)
         self.origin_repository.save(origin, valid_time=valid_time)
 
+        if not origin.result and origin.origin_type != OriginType.INFERENCE:
```
Comment (underdarknl) [inline octopoes/octopoes/core/service.py:194]:
```suggestion
        # origins that are stale need to be deleted. #3561 
        if not origin.result and origin.origin_type != OriginType.INFERENCE:
```

Comment (originalsouth) [reply octopoes/octopoes/core/service.py:195]: Trailing space

**Review by underdarknl — APPROVED** [review]

**Pull request merged**